### PR TITLE
Fixed jumplist description null on Android

### PR DESCRIPTION
**Pull request by HanamiM**
GitHub Issue (If applicable): #1418 

## PR Type

What kind of change does this PR introduce?

- Bugfix

## What is the current behavior?

Application crashes when JumpListItem description is null.

## What is the new behavior?

Application no longer crashes.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tested code with current [supported SDKs](https://github.com/unoplatform/uno/blob/master/README.md#uno-features)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/unoplatform/uno/tree/master/doc/ReleaseNotes)
- [x] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Comment by MartinZikmund** [comment]
@HanamiM Hi! Thank you for the fix, please add a simple unit test for this as well 😉 .

File: src/Uno.UI.RuntimeTests/Tests/Windows_UI_StartScreen/Given_JumpListItem.cs [hunk 1]
```diff
@@ -111,6 +111,29 @@ public void When_Logo_Uri_Is_ms_appx()
 			item.Logo = new Uri("ms-appx:///Test.png");
 			Assert.AreEqual("ms-appx:///Test.png", item.Logo.AbsoluteUri);
 		}
+
+		[TestMethod]
+		public async Task When_Description_Is_Empty_No_Exception()
+		{
+			try
```
Comment (MartinZikmund) [inline src/Uno.UI.RuntimeTests/Tests/Windows_UI_StartScreen/Given_JumpListItem.cs]: Test will need to run on the UI thread

Comment (jeromelaban) [reply src/Uno.UI.RuntimeTests/Tests/Windows_UI_StartScreen/Given_JumpListItem.cs]: See this for an example:

https://github.com/unoplatform/uno/blob/77e733e95e6e198b90eb24510e0225dfc8979fea/src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_TextBlock.cs#L33-L35

Comment (HanamiM) [reply src/Uno.UI.RuntimeTests/Tests/Windows_UI_StartScreen/Given_JumpListItem.cs]: Fixed to run on UI thread

**Comment by carldebilly** [comment]
@HanamiM Are you able to change your commit messages to follow Conventional Commits? (_squashing_)

**Pull request closed without merging**